CLI/EDDBFormatter/__main__.py

```diff
@@ -36,6 +36,14 @@ def run_eddb():
         print("Roger, aborting...")
         sys.exit()
 
+    distance = input(
+        "Do you want to filter the list to only stations within a certain range? (800 LS) (Y/n) "
+    )
+    if distance.upper() != "Y":
+        filter_dist = False
+    else:
+        filter_dist = True
+
```

Comment:
```py
filter_dist = False if dist.upper() != "Y" else True
```

Reply:
1. use casefold
2. this if statement can be inlined..
```suggestion
    filter_dist =  distance.casefold() == "y"
```